modules/records.nix

```
@@ -299,8 +299,8 @@ lib.mapAttrs
               description = lib.mdDoc ''
                 The canonical hostname of the machine providing the service.
               '';
-              example = "ftp://example.com/public";
-              type = lib.types.int;
+              example = "example.com";
+              type = with lib.types; nullOr str; # change str to lib.types.domain once it exists;
```

```suggestion
              example = "example.com";
              type = lib.types.str; # change str to lib.types.domain once it exists;
```
nullOr shouldn't be needed here since if I understand correctly target is a required filed once you create a srv record and if no srv record is set it will just default to an empty attrset which should be fine, please tell me if I'm missing something.
I also wonder how I messed that up since the comment is actually correct :thinking:

---

Agree.  I mindlessly copied it when I went looking to see if there was a more specific type than `types.str`, but I can't think of null making any sense here.